### ISSUE TITLE
Cell drag drop broken in Linux Chrome v97

### ISSUE THREAD

**Issue by j-fu** [fonsp/Pluto.jl#1809]
[123.jl.txt](https://github.com/fonsp/Pluto.jl/files/7829013/123.jl.txt)

Hi,I think with chrome (linux) drag & drop ceased to work at all...
My suspicion is chrome v97, because the fix in #1780 ceased to work, and so it doesn't work in 0.17.5 as well. I also tried Pluto 0.16.1 for double checking - the same.

(funny: I could rename #1684 but not reopen it...) 

https://user-images.githubusercontent.com/9221937/148549638-4fb93da5-73dc-46df-a88a-a76803a49317.mp4

[123.jl.txt](https://github.com/fonsp/Pluto.jl/files/7829014/123.jl.txt)

**Comment by pankgeorg** [comment]
@j-fu it is also broken if you set zoom level to 100% (Control + 0)? (long shot from browsing [possibly related issues ](https://stackoverflow.com/questions/52407557/drag-and-drop-issue-in-chrome-related-to-windows-scale-125) on stackoverflow)

**Comment by j-fu** [comment]
Hi thanks for caring !  It used to work with older chrome versions. Tried to verify this, now the story goes as follows:
- deinstalled google-chrome
-  tried to find an older rpm version  ( I am on opensuse...) to verify, the newest of those was v87, it seemed to be too old for pluto
-  Ok, then why not moving forward ? Installed  chrome-98-beta - there it works.
-  Back to stable (chrome-97-stable): after fresh install there it works as well. Either they patched this after my last system update or something else was rotten after too much of surfing...

Long story short: I think we can close this, and if it occurs again the first hint would be just to reinstall chrome ...
